manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: GeorgieLabs.SoundWireServer
+PackageVersion: 2.5.0

```

```suggestion
PackageVersion: "2.5"
```
Please update file name and path to match this change.